Code/Tools/SerializeContextTools/Utilities.h

```diff
@@ -35,11 +35,10 @@ namespace AZ
 
             static AZStd::vector<AZ::Uuid> GetSystemComponents(const Application& application);
 
-            static bool InspectSerializedFile(
-                const char* filePath,
-                SerializeContext* sc,
-                const ObjectStream::ClassReadyCB& classCallback,
-                Data::AssetFilterCB assetFilterCallback = AZ::Data::AssetFilterNoAssetLoading);
+            //! Converts path into a path that's relative to the highest-priority containing folder
+            //! (the Assets folder for a Gem or the Projects folder for project's assets)
+            static AZStd::string GenerateRelativePosixPath(const AZStd::string& projectPath, const AZStd::string& path);
+            static AZStd::string GetStringAfterFirstOccurenceOf(const AZStd::string& toFind, const AZStd::string& string);
```

Comment:
Since this function are only returning a substring of the input, a string_view can be accepted and returned, which avoids the need to allocate a new string due to using `AZStd::string::substr`.
(`AZStd::string_view::substr` just returns a new view into string data).
```suggestion
            static AZStd::string_view GetStringAfterFirstOccurenceOf(AZStd::string_view toFind, AZStd::string_view string);
```